File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.java
import java.io.*;
import java.util.*;

public class Main {
    private static int N;

    public static void main(String[] args) {
        FastReader scan = new FastReader();
        N = scan.nextInt();

        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < N; i++) {
            int a = scan.nextInt();
            int b = scan.nextInt();
            sb.append(a + b).append("\n");
        }

        System.out.println(sb);
    }

    static void input() {
    }

    static class FastReader {
        private final BufferedReader br;
        private StringTokenizer st;

        public FastReader() {
            this.br = new BufferedReader(new InputStreamReader(System.in));
        }

        String nextString() {
            while (st == null || !st.hasMoreElements()) {
                try {
                    st = new StringTokenizer(br.readLine());
                } catch (IOException e) {
                    throw new RuntimeException(e);
                }
            }
            return st.nextToken();
        }

        int nextInt() {
            return Integer.parseInt(nextString());
        }

        long nextLong() {
            return Long.parseLong(nextString());
        }

        String nextLine() {
            String str = "";
            try {
                str = br.readLine();
            } catch (IOException e) {
                throw new RuntimeException(e);
            }
            return str;
        }
    }
}

File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.kt
import java.io.*

private var N = 0
private var a = 0
private var b = 0

fun main() {
    val sb = StringBuilder()

    BufferedReader(InputStreamReader(System.`in`)).use {
        val scan = FastReader(it)

        N = scan.nextInt()
        repeat(N) {
            a = scan.nextInt()
            b = scan.nextInt()
            sb.append(a + b).appendLine()
        }
    }

    println(sb)
}

fun input() {
    BufferedReader(InputStreamReader(System.`in`)).use {
        val scan = FastReader(it)
    }
}

class FastReader(private val reader: BufferedReader) {
    private val tokenizer = StreamTokenizer(reader)

    fun nextString(): String {
        tokenizer.nextToken()
        return tokenizer.sval
    }

    fun nextInt(): Int {
        tokenizer.nextToken()
        return tokenizer.nval.toInt()
    }

    fun nextLine(): String {
        return reader.readLine()
    }
}